registry/migrations/env.py

```diff
@@ -57,7 +57,7 @@ def run_migrations_online():
 
     # this callback is used to prevent an auto-migration from being generated
     # when there are no changes to the schema
-    # reference: http://alembic.readthedocs.org/en/latest/cookbook.html
+    # reference: http://alembic.zzzcomputing.com/en/latest/cookbook.html
```

Comment:
sorry, why this change?

Reply:
I re-generated all of the migrations; I guess the new version of alembic did that. The old page doesn't exist anymore, so this looks right.